lua/auto-dark-mode/init.lua

```diff
@@ -27,7 +30,13 @@ local function parse_query_response(res)
 		-- 0: no preference
 		-- 1: dark
 		-- 2: light
-		return string.match(res, "uint32 1") ~= nil
+		if string.match(res, "uint32 1") ~= nil then
+			return true
+		elseif string.match(res, "unit32 2") ~= nil then
```

Comment:
this can never be true because of a typo: `unit32 2` when it should be `uint32 2`.